pkg/controller/datavolume-controller.go

```diff
@@ -298,45 +340,74 @@ func (c *DataVolumeController) syncHandler(key string) error {
 	return nil
 }
 
-func (c *DataVolumeController) updateImportStatusPhase(pvc *corev1.PersistentVolumeClaim, dataVolumeCopy *cdiv1.DataVolume) {
+func (c *DataVolumeController) updateImportStatusPhase(pvc *corev1.PersistentVolumeClaim, dataVolumeCopy *cdiv1.DataVolume) (eventtype, reason, message string) {
```

Comment:
I am fairly certain the linter will fail the name eventtype, I think it should be eventType.